core/lib/libtomcrypt/acipher_helpers.h

```diff
@@ -35,7 +35,7 @@ static inline TEE_Result convert_ltc_verify_status(int ltc_res, int ltc_stat)
 	}
 }
 
-#ifdef CFG_CRYPTOLIB_NAME_tomcrypt
+#ifdef _CFG_CORE_LTC_ECC
```

Comment:
This is not strictly related to 25519 support, please make a separate patch for this fix.